hisiodetbt.src

```diff
@@ -0,0 +1,182 @@
+;  highsiodet.src - check for highspeed drives
```

Comment:
can you change that to match the filename and add a short note that it applies only to BT?

Reply:
Sure